storage/oasis/consensus.go

```diff
@@ -0,0 +1,357 @@
+package oasis
```

Comment:
er why `oasis`? do we expect to have non-oasis data sources?

Reply:
Maybe not anymore. But initially, we thought we'd have different data sources (metadata to supplement indexer data from the [metadata registry](https://github.com/oasisprotocol/metadata-registry), for example) so hence the name.